### ISSUE TITLE
Reference issue from pull request

### ISSUE THREAD

**Issue by svarlamov** [go-gitea/gitea#3134]
- Gitea version (or commit ref): 7ec6cdd 

## Description

When referencing an issue from a pull request description, ie., `targeting #125` links properly, but when you're in the issue, there's no message saying that it was referenced in a pull request (such as Gitea does with commit messages or the way that Github does with pull request descriptions)

**Comment by svarlamov** [comment]
Also, referencing an issue, ie., `see #125` from an issue, links, but when you're in the referenced issue, there's no message about the reference -- as you would have in Github for instance

**Comment by lafriks** [comment]
Can you create sample on try.gitea.io to demonstrate this problem

**Comment by svarlamov** [comment]
@lafriks - sure!

Issue to issue ref:

1: https://try.gitea.io/svarlamov/sample-issues-3134/issues/1
2: https://try.gitea.io/svarlamov/sample-issues-3134/issues/2

2 references 1, but there's nothing in 1 to show that (as there would be for a commit msg)

PR to issue ref:

PR(a): https://try.gitea.io/svarlamov/sample-issues-3134/pulls/3
ISSUE(b): https://try.gitea.io/svarlamov/sample-issues-3134/issues/1

PR(a) references ISSUE(b), however, there's nothing to indicate that in ISSUE(b) -- as there otherwise would be for a commit msg

**Comment by lunny** [comment]
I think @svarlamov means reference comments.

**Comment by funkyfuture** [comment]
i think he means an information like the fourth under [this comment](https://github.com/go-gitea/gitea/issues/3430#issuecomment-361303846) in a pull request here.

**Comment by funkyfuture** [comment]
and like the one below that. :-)

**Comment by stevegt** [comment]
I've created #4302 to address the case of the issue backlinks.  I'm pretty sure PR backlinks (this issue) could use the same table discussed in #4302, if that turns out to be the solution.

**Comment by AfroThundr3007730** [comment]
@lunny Perhaps the stale bot should be configured to ignore issues that are tracked in a milestone, or another tag could be created to exempt an issue from auto-closing.

It's not exactly ideal to keep bumping them manually.

**Comment by lafriks** [comment]
there is already label `reviewed/confirmed` that stalebot won't touch :)

**Comment by raucao** [comment]
+1 for linking back to the PR from the issue view.

But also, it'd be nice to be able to close issues from PR descriptions. I.e. if the reference is "closes #123" it would close issue number 123 once the PR is merged (same as from commits). I do this all the time on GitHub and kind of miss it on Gitea now.

**Comment by guillep2k** [comment]
This is pending. I'm interested in this feature, I might work on it next. Any caveats I should be aware of?

**Comment by Worthy7** [comment]
@skddc One thing at a time ;-) make a new issue on that one (It's probably somewhere already!) please

@guillep2k  Thanks!
